### ISSUE TITLE
Add "Security Considerations"

### ISSUE THREAD

**Issue by dret** [dret/I-D#82]
currently just a placeholder in the draft; needs to be completed before last call. probably some language about the fact that by definition, out-of-line links should be treated with care because they are not part of the resource they link.

**Comment by dret** [comment]
link to RFC8288 (only consideration that are not header-specific), also talk about:
- context issues (interpretation of relative URIs)
- trust issues around links (3rd party links scenario)

**Comment by dret** [comment]
@hvdsomp, can you please briefly look at the security considerations and give feedback? if they're good enough, next on the list is #126 which depends a bit on the security considerations. thanks.

**Comment by dret** [comment]
done.
